.rodare.json

```diff
@@ -0,0 +1,37 @@
+{
+    "creators": [
+        {
+            "name": "Widera, René",
+            "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
+            "orcid": "0000-0003-1642-0459"
+        },
+        {
+            "name": "Huebl, Axel",
+            "orcid": "0000-0003-1943-7141"
```

Comment:
Why didn't you put his affiliations?

Reply:
I do not know it^^

Reply:
The question is also should it be the old or new.

Reply:
talked to axel it must be the old affiliatuon but can be optionally be extendet with "... now XYZ" but than we need to track the contributor so we should stay with the affiliarion from the date where the user contributed.